### PR TITLE
feat(frontend): new style for settings page

### LINKED ISSUE #2519: 🚀 [Feature] [Design] New Design for Settings
## 🚀 Feature

Implement the new Design for the settings page, also consider the split between Profile & Settings.

### PR DESCRIPTION
**Pull request by ogerly**
## 🍰 Pullrequest
new style for settings page, remove unused components
- dark style modus in settings

### Issues
- fixes #2519 

https://github.com/gradido/gradido/assets/1324583/582ac073-b99e-401e-92a3-9f5df08e6169

### CODE REVIEW

**Comment by ogerly** [comment]
image

File: frontend/src/components/UserSettings/UserPassword.vue [hunk 1]
```diff
@@ -103,7 +103,7 @@ export default {
     disabled() {
       return !(
         this.form.newPassword.password !== '' &&
-        this.form.newPassword.password === this.form.newPassword.passwordRepeat
+        this.form.newPassword.password !== this.form.newPassword.passwordRepeat
       )
     },
```
Comment (mahula) [inline frontend/src/components/UserSettings/UserPassword.vue:108]: The enabling behavior of the submit button does not work as expected:
Expected: after the new password has been enteredand been validated, the button should be enabled.

Current behavior:
When entering the new password the first time, the button gets enabled.
When entering the new password the second time,the button gets disabled.

**Review by mahula — CHANGES_REQUESTED** [review]

File: frontend/src/components/Inputs/InputUsername.spec.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    })
+    it('renders the component', () => {
+      expect(wrapper.find('div.input-username').exists()).toBeTruthy()
+    })
+
+    describe('currentValue', () => {
+      beforeEach(async () => {
+        wrapper = Wrapper()
+        await wrapper.setProps({ value: 'petra' })
+        wrapper.vm.currentValue = 'petra swastiska'
```
Comment (Elweyn) [inline frontend/src/components/Inputs/InputUsername.spec.js:39]: Should be done on the input field and not mock it.
Something like this:
```suggestion
        await wrapper.find('div.input-username.input[type="text"]').setValue('petra swastiska')
```

File: frontend/src/components/UserSettings/UserName.spec.js [hunk 3]
```diff
@@ -39,41 +39,49 @@ describe('UserName Form', () => {
...
-      it('has a submit button', () => {
-        expect(wrapper.find('button[type="submit"]').exists()).toBeTruthy()
+    //   it('has no component username change ', () => {
+    //     expect(wrapper.findComponent({ name: 'InputUsername' }).exists()).toBeTruthy()
+    //   })
+    // })
+
+    describe('edit username', () => {
+      beforeEach(async () => {
+        wrapper.vm.username = 'petra'
```
Comment (Elweyn) [inline frontend/src/components/UserSettings/UserName.spec.js:80]:
```suggestion
        wrapper.findComponent({ name: 'InputUsername' }).setValue('petra')
```

File: frontend/src/pages/Settings.spec.js [hunk 4]
```diff
@@ -1,42 +1,133 @@
...
-    it('has a user change newsletter form', () => {
-      expect(wrapper.findComponent({ name: 'UserNewsletter' }).exists()).toBeTruthy()
+    describe('darkMode style', () => {
+      it('default darkMode is true', () => {
+        expect(wrapper.vm.darkMode).toBe(true)
+      })
+
+      describe('sets dark mode ', () => {
+        beforeEach(() => {
+          wrapper.vm.darkMode = false
```
Comment (Elweyn) [inline frontend/src/pages/Settings.spec.js:121]: please use wrapper.find and click

File: frontend/src/pages/Settings.spec.js [hunk 5]
```diff
@@ -1,42 +1,133 @@
...
+            updateUserInfos: {
+              validValues: 3,
+            },
+          },
+        })
+      })
+
+      it('Cange first and lastname', async () => {
+        wrapper.find('[data-test="test-firstname"]')
+        wrapper.find('[data-test="lastname"]')
```
Comment (Elweyn) [inline frontend/src/pages/Settings.spec.js:86]: please use this line 72 and 73

**Review by Elweyn — CHANGES_REQUESTED** [review]
Some small changes on the unit the rest seem great!

File: frontend/src/pages/Settings.spec.js (hunk 5 above)
Comment (ogerly) [reply frontend/src/pages/Settings.spec.js:86]: done

File: frontend/src/pages/Settings.spec.js (hunk 4 above)
Comment (ogerly) [reply frontend/src/pages/Settings.spec.js:121]: done

**Review by Elweyn — APPROVED** [review]

**Pull request merged**